packages/uxpin-merge-cli/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@uxpin/merge-cli",
-  "version": "3.0.5",
+  "version": "3.0.5-debug-mode",
```

Comment:
Please correct me if I am wrong, but should `-debug-mode` be removed from the version? If we plan to release this as a latest version of the CLI?